stories/components/ButtonDropdown.stories.mdx

```diff
@@ -0,0 +1,63 @@
+
+import { Canvas, Meta, Story, ArgsTable, Source } from '@storybook/addon-docs';
+import ButtonDropdown from '@shell/components/ButtonDropdown';
+
+<Meta 
+  title="Components/ButtonDropdown" 
+  component={ButtonDropdown}
+/>
+
+export const Template = (args, { argTypes }) => ({
+  components: { ButtonDropdown },
+  props:      Object.keys(argTypes),
+  template:   '<ButtonDropdown v-bind="$props" />',
+})
+
+# Button Dropdown
+
+Dropdowns are toggleable lists of links or actions which are displayed by clicking on them.
+
+### Description
```

Comment:
Could you please add also about the `hash`:
- brief information about the functionality
- requirements: `vue-router` or `Nuxt`